geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -113,6 +113,7 @@
   private volatile boolean _isStopped = true;
 
   private volatile long waitForReAuthenticationStartTime = -1;
+  private final Object re_auth_lock = new Object();
```

Comment:
Lots of the existing code in this area uses underscores but you really should name it something better without them. "reauthenticationLock" would be a good name.

Reply:
done